test/powershell/Modules/Microsoft.PowerShell.Core/assets/CompatiblePSEditionsModules/BothModule/BothModule.psd1

```diff
@@ -0,0 +1,122 @@
+#
```

Comment:
These test modules are pretty basic.  I wonder if it's better to just generate these at runtime as test variations?